aem-authoring-toolkit-plugin/src/test/resources/widget/accordion/_cq_dialog.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+          jcr:primaryType="nt:unstructured"
+          sling:resourceType="cq/gui/components/authoring/dialog"
+          jcr:title="Accordion Test Dialog"
+          width="{Double}560.0"
+          height="{Double}480.0">
+    <content jcr:primaryType="nt:unstructured"
+             sling:resourceType="granite/ui/components/coral/foundation/container">
+        <items jcr:primaryType="nt:unstructured">
+            <accordion
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/accordion">
+                <items jcr:primaryType="nt:unstructured">
+                    <basic
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Basic"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <parentConfig jcr:primaryType="nt:unstructured"
+                                      disabled="{Boolean}true"
+                                      active="{Boolean}false"/>
+                        <items jcr:primaryType="nt:unstructured">
+                            <buttonField1 jcr:primaryType="nt:unstructured"
```

Comment:
(See above) please remove as much detail as possible from the sample XML. We are not testing button here, only the container)

Reply:
Fixed